templates/nextflow-ecs-task-definition.yaml

```diff
@@ -173,16 +234,33 @@ Resources:
               Value: !Ref 'TowerLicense'
             - Name: FLYWAY_LOCATIONS
               Value: classpath:db-schema/mysql
+            - Name: TOWER_GITHUB_CLIENT
+              Value: !Ref TowerGithubClientId
+            - Name: TOWER_GITHUB_SECRET
+              Value: !Ref TowerGithubSecret
           EntryPoint:
             - /bin/sh
           Command:
             - -c
             - /tower.sh
-  TowerService:
-    Type: AWS::ECS::Service
-    Properties:
-      Cluster: !Ref 'ClusterName'
-      DesiredCount: 1
-      ServiceName: TowerService
-      TaskDefinition: !Ref 'TowerTask'
-      LaunchType: EC2
+          MountPoints:
+            - ContainerPath: '/tower.yml'
```

Comment:
`tower.yml` appears in other files and in this file a few times.  would be nice to make it a parameter so you can make it consistent.

Reply:
Good suggestion -- I will do this in the follow up PR for a branch already in progress that adds EFS